_maps/map_files/Pillar_of_Spring/TGS_Pillar_of_Spring.dmm

```diff
@@ -0,0 +1,9245 @@
+/obj/item/clothing/under/liaison_suit/formal,
+/obj/item/clothing/under/liaison_suit,
+/obj/item/clothing/under/liaison_suit/outing,
+/obj/item/clothing/under/liaison_suit/suspenders,
+/obj/item/clipboard{
+	pixel_x = 5
+	},
+/obj/item/clothing/suit/armor/bulletproof,
+/obj/item/attachable/suppressor,
+/obj/item/clothing/tie/holster,
+/turf/open/floor/carpet{
+	icon_state = "carpetside";
+	dir = 9
+	},
+/area/mainship/command/corporateliaison)
+"iT" = (
+/turf/open/floor/mainship/research/containment/floor2{
+	dir = 5
+	},
+/area/mainship/medical/medical_science)
+"iU" = (
+/turf/closed/wall/mainship/research/containment/wall/purple{
+	icon_state = "containment_window";
+	dir = 1
+	},
+/area/mainship/medical/medical_science)
+"iV" = (
+/turf/open/floor/mainship{
+	dir = 8;
+	icon_state = "sterile_green_side"
+	},
+/area/mainship/medical/medical_science)
+"iW" = (
+/obj/machinery/disposal,
+/obj/structure/disposalpipe/trunk{
+	dir = 4
+	},
+/turf/open/floor/wood,
+/area/mainship/command/corporateliaison)
+"iX" = (
+/turf/open/floor/mainship/mono,
+/area/mainship/command/airoom)
+"iY" = (
+/obj/item/bedsheet/blue,
+/obj/effect/landmark/start/marine/corporateliaison,
+/obj/structure/bed{
+	desc = "For prime comfort.";
+	name = "fancy bed"
+	},
+/obj/item/toy/farwadoll{
+	pixel_x = 5
+	},
+/turf/open/floor/carpet{
+	icon_state = "carpetside";
+	dir = 1
+	},
+/area/mainship/command/corporateliaison)
+"iZ" = (
+/obj/structure/disposalpipe/segment{
+	icon_state = "pipe-c";
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden{
+	dir = 1
+	},
+/obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer1{
+	dir = 1
+	},
+/obj/structure/cable{
+	d1 = 2;
+	d2 = 8;
+	icon_state = "2-8"
+	},
+/obj/structure/cable{
+	d1 = 2;
+	d2 = 4;
+	icon_state = "2-4"
+	},
+/turf/open/floor/mainship,
+/area/mainship/hallways/aft_hallway)
+"ja" = (
+/obj/structure/dropship_equipment/sentry_holder,
+/obj/effect/decal/warning_stripes/thin{
+	dir = 8
+	},
+/obj/effect/decal/warning_stripes/thin{
+	dir = 1
+	},
+/turf/open/floor/mainship{
+	dir = 8;
+	icon_state = "orange"
+	},
+/area/mainship/hallways/hangar)
+"jb" = (
+/turf/open/floor/mainship/mono,
+/area/mainship/command/self_destruct)
+"jc" = (
+/obj/structure/largecrate/random/case/double,
+/turf/open/floor/mainship/mono,
+/area/mainship/command/self_destruct)
+"jd" = (
+/turf/open/shuttle/escapepod{
+	icon_state = "floor1"
+	},
+/area/mainship/command/self_destruct)
+"jf" = (
+/obj/structure/bed/chair/office/dark{
+	dir = 4
+	},
+/turf/open/floor/mainship{
+	dir = 8;
+	icon_state = "emerald"
+	},
+/area/mainship/command/cic)
+"jg" = (
+/obj/structure/dropship_equipment/electronics/targeting_system,
+/obj/effect/decal/warning_stripes/thin{
+	dir = 1
+	},
+/obj/machinery/light{
+	dir = 4
+	},
+/turf/open/floor/mainship{
+	dir = 4;
+	icon_state = "orange"
+	},
+/area/mainship/hallways/hangar)
+"jh" = (
+/obj/effect/decal/warning_stripes/thin{
+	dir = 6
+	},
+/obj/effect/decal/warning_stripe
```

Comment:
Subtype (feel free to remove the dir-based variants, all I care about is the id and name vars on these)